packages/adp-tooling/src/writer/options.ts

```diff
@@ -110,7 +110,8 @@ function addFioriToolsMiddlwares(ui5Config: UI5Config, config: AdpWriterConfig)
             url: config?.ui5?.frameworkUrl,
             version: config?.ui5?.minVersion ?? '' //default to latest if version is not set
         },
-        backend: [backendConfig as FioriToolsProxyConfigBackend]
+        backend: [backendConfig as FioriToolsProxyConfigBackend],
+        afterMiddleware: 'fiori-tools-preview'
```

Comment:
I don't object that change but can you explain why this is a problem? I am afraid we are not understanding the root cause here.

Reply:
Readme updated in: [619ae1e](https://github.com/SAP/open-ux-tools/pull/2146/commits/619ae1e4d64848291997270a4134034d24b97707)